src/test/java/amazoncucumber/Stepdefs.java

```diff
@@ -0,0 +1,94 @@
+package amazoncucumber;
+
+import cucumber.api.PendingException;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import cucumber.api.java.After;
+import org.junit.Assert;
+import cucumber.api.java.Before;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import pages.AmazonProductPage;
+import pages.AmazonResultPage;
+import pages.AmazonStartPage;
+import pages.BasePage;
+
+public class Stepdefs {
+    public static WebDriver webDriver;
+    AmazonResultPage resultPage;
+
+    @Before
+    public void setup() {
+        System.setProperty("webdriver.chrome.driver", "/usr/bin/chromedriver"); // Linux: /usr/bin/chromedriver MacOS: /Users/bit/bin/chromedriver
+        this.webDriver = new ChromeDriver();
+        webDriver.get("https://www.amazon.de");
+    }
+
+    @Given("User is on Amazon starting page")
```

Comment:
Why is in the regex here no starting and ending delimiter like "^$" but all the other steps have it? 
Find out what it does and when to use it (or not).